Umbraco-Cloud/Deployment/Content-Transfer/index.md

```diff
@@ -56,7 +56,7 @@ For Deploy to handle Forms data as content, you'll need to add the following set
 
 #### Umbraco version 9 and above
 
-For Deploy to handle Forms data as content, you'll need to ensure the `TransferFormsAsContent` setting is set to `true`:
+For Deploy to handle Forms data as content, you'll need to ensure the `TransferFormsAsContent` setting is set to `true` in the `appsettings.json`file:
```

Comment:
The sentence was being a bit wordy. How about something like this instead? 🤔 

```suggestion
For Deploy to handle Forms data as content, the `TransferFormsAsContent` setting found in the `appsettings.json` file needs to be `true`:
```

Reply:
Looks good @sofietoft 😄